PractiseMay11/PlantsTask/Gallery.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PractiseMay11.PlantsTask
+{
+    internal class Gallery
+    {
+        public Plant[] plants;
+
+        public Worker worker;
+
+        public Gallery(Plant[] plants, Worker worker)
+        {
+            this.plants = plants;
+            this.worker = worker;
+        }
+
+        public void Execute()
+        {
+            Random random = new Random(10);
+            foreach (var item in plants)
+            {
+                worker.Care(random.Next(1, 10), random.Next(1, 10), item);
+                if (item is Flower)
```

Comment:
Do you remember a simpler way for casting and variable declaration/initialisation?
item is Flower flower